src/Components/Browser.JS/src/Platform/Circuits/AutoReconnectCircuitHandler.ts

```diff
@@ -2,13 +2,16 @@ import { CircuitHandler } from './CircuitHandler';
 import { UserSpecifiedDisplay } from './UserSpecifiedDisplay';
 import { DefaultReconnectDisplay } from './DefaultReconnectDisplay';
 import { ReconnectDisplay } from './ReconnectDisplay';
+import { ILogger, LogLevel } from '../Logging/ILogger';
 export class AutoReconnectCircuitHandler implements CircuitHandler {
-  static readonly MaxRetries = 5;
-  static readonly RetryInterval = 3000;
-  static readonly DialogId = 'components-reconnect-modal';
-  reconnectDisplay: ReconnectDisplay;
+  public static readonly MaxRetries = 5;
+  public static readonly RetryInterval = 3000;
+  public static readonly DialogId = 'components-reconnect-modal';
+  public reconnectDisplay: ReconnectDisplay;
+  public logger: ILogger;
```

Comment:
Why make all these public? If it's for configurability, I think it should be done on a per-instance basis not via statics.

Reply:
These are changes are about making eslint/jest happy. ESlint complained about the lack of modifier and when I tried private, jest complained about it being private, so no other option but to make them public.

Right now there's no way for a customer to access AutoReconnectCircuitHandler, so I don't think it's a big deal.

Reply:
OK, my mistake. I was falsely assuming C# defaults (as in, private unless specified as public). Since this is TypeScript, it's public unless specified as private, so they were already public and your change is just stylistic.